src/main/java/me/jellysquid/mods/lithium/mixin/entity/slimechunk_cache/ChunkSerializerMixin.java

```diff
@@ -0,0 +1,36 @@
+package me.jellysquid.mods.lithium.mixin.entity.slimechunk_cache;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import me.jellysquid.mods.lithium.common.chunk.SlimeChunkStorage;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.structure.StructureManager;
+import net.minecraft.util.math.ChunkPos;
+import net.minecraft.world.ChunkSerializer;
+import net.minecraft.world.chunk.Chunk;
+import net.minecraft.world.poi.PointOfInterestStorage;
+import net.minecraft.server.world.ChunkHolder;
+
+@Mixin(ChunkSerializer.class)
+public class ChunkSerializerMixin {
+    @Inject(at = @At("RETURN"), method = "deserialize")
+    private static void cacheSlimeChunk(ServerWorld world, StructureManager structureManager, PointOfInterestStorage poiStorage, ChunkPos pos, CompoundTag tag, CallbackInfo ci) {
```

Comment:
Could probably avoid serialization of this altogether by recomputing in the ChunkStatus load task

Reply:
I don't see why serialization here is a bad thing other than it means you don't have to make sure it has been serialized before.